# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/telemetry/other/UntrustedModulesDataSerializer.cpp
# Commit: 6b399daabb18
# Full Hash: 6b399daabb187b957004361b4e53c43afc61e420
# Author: Nazım Can Altınova <canaltinova@gmail.com>
# Date: 2024-11-26 09:36:10
# Regressor Bug: 1634785
# File Overlap Count: 1
# Description:
#   Bug 1634785 - Rename BaseProfilerSharedLibraries.h to SharedLibraries.h r=mstange,profiler-reviewers
#   
#   Now that we only have one implementation of this code, we don't have to
#   differentiate it with "BaseProfiler".
#   
# ==============================================================================

diff -r 8ad3b7e6de23 -r 6b399daabb18 toolkit/components/telemetry/other/UntrustedModulesDataSerializer.cpp
--- a/toolkit/components/telemetry/other/UntrustedModulesDataSerializer.cpp	Mon Nov 25 22:09:05 2024 +0000
+++ b/toolkit/components/telemetry/other/UntrustedModulesDataSerializer.cpp	Mon Nov 25 22:09:05 2024 +0000
@@ -14,7 +14,7 @@
 #include "nsITelemetry.h"
 #include "nsUnicharUtils.h"
 #include "nsXULAppAPI.h"
-#include "BaseProfilerSharedLibraries.h"
+#include "SharedLibraries.h"
 
 namespace mozilla {
 namespace Telemetry {